arbeitszeit_web/request.py

```diff
@@ -16,8 +16,16 @@ def get_request_target(self) -> str: ...
 
 
 class QueryString(Protocol):
-    def get(self, key: str) -> Optional[str]:
+    def get(self, key: str) -> list[str]:
         """Get the value for a particular query arguments."""
```

Comment:
this could maybe also be updated with sth like
"Get the list of values for a query argument or [] if key does not exist." ?